example/centos/pipeline/build.yaml

```diff
@@ -1,21 +1,12 @@
 name: build
-# source-epoch will support reproducible builds:
-# https://reproducible-builds.org/docs/source-date-epoch/
-source-epoch: ${source_epoch}
 runner: "org.osbuild.centos9"
 stages:
-  - otk.external.osbuild_depsolve_dnf4:
-    architecture: ${architecture}
-    releasever: ${version}
-    module_platform_id: platform:el${version}
-    docs: ${packages.build.docs}
-    weak: ${packages.build.weak}
-    repositories: ${packages.repositories}
-    gpgkeys: ${packages.keys}
-    packages: ${packages.buildroot.packages}
+  - otk.external.osbuild-make-depsolve-dnf4-rpm-stage:
+      packageset: ${packages.build}
+      gpgkeys:
+        otk.include: "../common/gpgkeys.yaml"
   - type: org.osbuild.selinux
     options:
       file_contexts: etc/selinux/targeted/contexts/files/file_contexts
```

Comment:
We've never had a need to build an image with any other policy (except maybe ISOs?) but it'll be interesting to think about how this would work here.  At some point (not in this PR), we might need to do something to define `selinux_policy="targeted"`, which will affect the path here as well as the package selection for the rpm stage.

And if that's never needed for the selinux policy, the same need will probably arise for something else.  Do we have an idea of how we might do this in a clean way?